### PR TITLE
New ci/cd workflows based on tripper

### PR DESCRIPTION
**Pull request by francescalb**
# Description:
The old workflows where quite outdated  and are now based on the ci/cd package on sintef https://github.com/SINTEF/ci-cd

## Type of change:

- [ ] Bug fix.
- [ ] New feature.
- [ ] Documentation update.

## Checklist for the reviewer:

This checklist should be used as a help for the reviewer.

- [ ] Is the change limited to one issue?
- [ ] Does this PR close the issue?
- [ ] Is the code easy to read and understand, including clearly named variables?
- [ ] Do all new feature have an accompanying new test?
- [ ] Has the documentation been updated as necessary?

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,12 +1,101 @@
...
+authors = [
+    {name = "SINTEF", email = "Team4.0@SINTEF.no"},
+]
+description = "DLite plugin for Open Translation Environment (OTE) API."
+readme = "README.md"
+license = {file = "LICENSE"}
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
```
Comment (jesper-friis) [inline pyproject.toml:16]: Python 3.11 can be added, since we are already testing for it. 
We should also add Python 3.12, but that can be in a new PR.

**Review by jesper-friis — COMMENTED** [review]
Impressive update!

Do you know why some tests do not run?

File: pyproject.toml [hunk 2]
```diff
@@ -1,12 +1,101 @@
...
+    "tripper>=0.1.2",
+]
+
+[project.optional-dependencies]
+docs = [
+    "mike~=2.1",
+    "mkdocs>=1.5.3,<1.7",
+    "mkdocs-awesome-pages-plugin~=2.9",
+    "mkdocs-material>=9.5.5,<10",
+    "mkdocstrings[python]~=0.25.1",
```
Comment (jesper-friis) [inline pyproject.toml:39]: Shouldn't these be replaced with `==`?

Comment (francescalb) [reply pyproject.toml:39]: ~= means this version or higher minor versions. It is fine

File: pyproject.toml (hunk 1 above)
Comment (francescalb) [reply pyproject.toml:16]: 3.12 fails, so not yet.

File: pyproject.toml (hunk 2 above)
Comment (jesper-friis) [reply pyproject.toml:41]: Not that important, but for generating documentation I think it is safer to be strict. But lets not spend  time on it now...

**Review by jesper-friis — APPROVED** [review]

**Pull request merged**